testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -179,6 +187,7 @@ public void testSparkReadingTrinoData(StorageFormat storageFormat, CreateMode cr
                 //", TIMESTAMP '2020-06-28 14:16:00.456' _timestamp " +
                 ", TIMESTAMP '2021-08-03 08:32:21.123456 Europe/Warsaw' _timestamptz " +
                 ", DATE '1950-06-28' _date " +
+                ", CAST('binary0/1values' AS VARBINARY) _binary " +
```

Comment:
some still missing here see https://iceberg.apache.org/spark-writes/ (Iceberg type to Spark type section).
E.g we should test if we can read UUID written by Trino as String via Spark.

Reply:
for uuid there is a separate test `testSparkReadTrinoUuid`